aptos-move/move-examples/fungible_asset/sources/fungible_asset.move

```diff
@@ -0,0 +1,377 @@
+/// This defines the fungible asset module that can issue fungible assets of any `FungibleSource` object. The source
+/// can be a token object or any object that equipped with `FungibleSource` resource.
+module fungible_asset::fungible_asset {
+    use aptos_std::smart_table::{Self, SmartTable};
+    use aptos_framework::object::{Self, Object, object_address, generate_transfer_ref, DeleteRef, generate_delete_ref};
```

Comment:
could we avoid importing functions directly?

Reply:
sry, what's the benefit? Actually those are auto-imported by ide...

Reply:
I can try to find the rust recommendation, but generally speaking, reading code is a lot easier if you leave somewhat namespaced functions. We have also done this consistently within our code and framework.